content/logs/index.md

```diff
@@ -130,15 +133,59 @@ logs:
 
 {{< /highlight >}}
 
+### Filter logs
+
+All logs are not equals and you might want to send only a specific subset of logs to Datadog.
```

Comment:
"All logs are not *equal* and you *may* want to send only a specific subset of logs to Datadog."

Reply:
👍